<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="6" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="TestSuite" duration-ms="14722" started-at="2018-06-14T00:59:56Z" finished-at="2018-06-14T01:00:11Z">
    <groups>
      <group name="regression">
        <method signature="AddCustomerTest.addExistingCustomer()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="addExistingCustomer" class="com.test.task.QETestTask.AddCustomerTest"/>
        <method signature="AddCustomerTest.addNewCustomer()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="addNewCustomer" class="com.test.task.QETestTask.AddCustomerTest"/>
        <method signature="RemoveCustomerTest.removeExistingCustomer()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="removeExistingCustomer" class="com.test.task.QETestTask.RemoveCustomerTest"/>
        <method signature="RemoveCustomerTest.removeNonExistingCustomer()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="removeNonExistingCustomer" class="com.test.task.QETestTask.RemoveCustomerTest"/>
        <method signature="UpdateCustomerTest.updateExistingCustomer()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="updateExistingCustomer" class="com.test.task.QETestTask.UpdateCustomerTest"/>
        <method signature="UpdateCustomerTest.updateNonExistingCustomer()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="updateNonExistingCustomer" class="com.test.task.QETestTask.UpdateCustomerTest"/>
      </group> <!-- regression -->
    </groups>
    <test name="Test Task" duration-ms="14722" started-at="2018-06-14T00:59:56Z" finished-at="2018-06-14T01:00:11Z">
      <class name="com.test.task.QETestTask.AddCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="setUp" is-config="true" duration-ms="3051" started-at="2018-06-13T17:59:56Z" finished-at="2018-06-13T17:59:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="addExistingCustomer()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="addExistingCustomer" duration-ms="1411" started-at="2018-06-13T17:59:59Z" finished-at="2018-06-13T18:00:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addExistingCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="tearDown" is-config="true" duration-ms="85" started-at="2018-06-13T18:00:00Z" finished-at="2018-06-13T18:00:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="setUp" is-config="true" duration-ms="1077" started-at="2018-06-13T18:00:01Z" finished-at="2018-06-13T18:00:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="addNewCustomer()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="addNewCustomer" duration-ms="860" started-at="2018-06-13T18:00:02Z" finished-at="2018-06-13T18:00:02Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Customer details added to the database successfully expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Customer details added to the database successfully expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at com.test.task.QETestTask.AddCustomer.verifyCustomerAdded(AddCustomer.java:32)
	at com.test.task.QETestTask.AddCustomerTest.addNewCustomer(AddCustomerTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.task.QETestTask.AddCustomerTest@7c601d50]" name="tearDown" is-config="true" duration-ms="80" started-at="2018-06-13T18:00:02Z" finished-at="2018-06-13T18:00:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.task.QETestTask.AddCustomerTest -->
      <class name="com.test.task.QETestTask.UpdateCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="setUp" is-config="true" duration-ms="992" started-at="2018-06-13T18:00:03Z" finished-at="2018-06-13T18:00:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="updateExistingCustomer()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="updateExistingCustomer" duration-ms="783" started-at="2018-06-13T18:00:04Z" finished-at="2018-06-13T18:00:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateExistingCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="tearDown" is-config="true" duration-ms="83" started-at="2018-06-13T18:00:04Z" finished-at="2018-06-13T18:00:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="setUp" is-config="true" duration-ms="902" started-at="2018-06-13T18:00:04Z" finished-at="2018-06-13T18:00:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="updateNonExistingCustomer()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="updateNonExistingCustomer" duration-ms="900" started-at="2018-06-13T18:00:05Z" finished-at="2018-06-13T18:00:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateNonExistingCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.task.QETestTask.UpdateCustomerTest@79b2852b]" name="tearDown" is-config="true" duration-ms="78" started-at="2018-06-13T18:00:06Z" finished-at="2018-06-13T18:00:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.task.QETestTask.UpdateCustomerTest -->
      <class name="com.test.task.QETestTask.RemoveCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="setUp" is-config="true" duration-ms="984" started-at="2018-06-13T18:00:06Z" finished-at="2018-06-13T18:00:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="removeExistingCustomer()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="removeExistingCustomer" duration-ms="1407" started-at="2018-06-13T18:00:07Z" finished-at="2018-06-13T18:00:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeExistingCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="tearDown" is-config="true" duration-ms="79" started-at="2018-06-13T18:00:09Z" finished-at="2018-06-13T18:00:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="setUp" is-config="true" duration-ms="992" started-at="2018-06-13T18:00:09Z" finished-at="2018-06-13T18:00:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="removeNonExistingCustomer()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="removeNonExistingCustomer" duration-ms="711" started-at="2018-06-13T18:00:10Z" finished-at="2018-06-13T18:00:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Customer details were not removed since the customer does not exist expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Customer details were not removed since the customer does not exist expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at com.test.task.QETestTask.RemoveCustomer.verifyNonExistingCustomerRemoved(RemoveCustomer.java:31)
	at com.test.task.QETestTask.RemoveCustomerTest.removeNonExistingCustomer(RemoveCustomerTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeNonExistingCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.task.QETestTask.RemoveCustomerTest@326d27ac]" name="tearDown" is-config="true" duration-ms="89" started-at="2018-06-13T18:00:10Z" finished-at="2018-06-13T18:00:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.task.QETestTask.RemoveCustomerTest -->
    </test> <!-- Test Task -->
  </suite> <!-- TestSuite -->
</testng-results>
